### ISSUE TITLE
Homepage A11Y: Stats section screen reader compatability

### ISSUE THREAD

**Issue by josieg** [BCcampus/open-sage#144]
**Describe the bug**
The stats section is confusing for someone using a screen reader, and there are a few reasons for this and I'm not sure about what a good fix would be. Here are the problems I found:

- The numbers are headings (confusing for someone reading headings on a page)
- Only the icons are linked (confusing for someone reading links on the page) - plus all icons are linked to the same place so why do we need four links?
- The icons themselves are decorative and don't actually require link text

**To Reproduce**
1. Go to https://helga.bccampus.ca/wp/opentextbooks/
2. Read page with a screen reader. 
3. When reading headings, screen reader says, "Open Education at BCcampus heading 4, 32 heading 3, 149 heading 3, 14604 heading 3, $1459800-$1851366 heading 3, Getting Started heading 4"
4. When reading links/image descriptions, screen reader says "linked image icon representing an institution"

**Expected behavior**

- decorative images should not have alt tags
- numbers should not be marked as headings
- link text should be descriptive

My suggestion: Remove alt tags and links from icons, and remove heading status from numbers. Instead create one link (imitating the Browse All Guides and Toolkits button) that says "Find More Open Textbook Stats." Tagging @ehbeattie for approval re: style and any further thoughts.

**Screenshots**
![stats](https://user-images.githubusercontent.com/23706762/54724745-a0800e80-4b29-11e9-8a84-8f00c7393892.jpg)

**Comment by alex-418** [comment]
1. I can remove headings from numbers, no problemo 
2. Talked about this one with @ehbeattie and decided to keep the links 
3. I can remove headings from numbers, no problemo
4. @josieg can you please let us know what you expect these to say instead ?

**Comment by josieg** [comment]
To confirm, you are going to keep the links on the icons? If that's the case, the alt tag for those icons becomes the link text. Change alt text as follows:

- Stats for BC Institutions
- Stats for BC Faculty
- Stats for BC Students
- Stats for BC Savings

**Comment by alex-418** [comment]
Yes. Great, thanks @josieg will change the alt text

**Comment by alex-418** [comment]
Completed via https://github.com/BCcampus/open-sage/commit/1678ffd710b06b25791f7dfab30beed34fb05817

**Comment by ehbeattie** [comment]
Looks great!
